hendrix/deploy/base.py

```diff
@@ -281,10 +265,19 @@ def restart(self, fd=None):
 
     def disownService(self, name):
         """
-        disowns a service on hendirix by name
-        returns a factory for use in the adoptStreamPort part of setting up
+        disowns a service on hendrix by name
+        returns a factory for use in the adoptStreamConnection part of setting up
         multiple processes
         """
         _service = self.hendrix.getServiceNamed(name)
         _service.disownServiceParent()
         return _service.factory
+
+
+class Data(object):
```

Comment:
This name is too generic for my tastes.  All classes with attributes will yield instances which hold "data."  What is this class *really*?